Write createReviews script to seed databases

willmainwaring opened the pull request.

Commit:
move mySQL-related files to sub-folder

Commit:
write createReviews and seed scripts

Commit:
move images to aws and randomly generate image URL

Commit:
modify review script for both databases

Commit:
remove unused id column

keenanthomson reviewed (commented):
This looks good to me - I think it's mergeable, but two questions:

- could you reduce your createReviews to a single script? (ie. both cassandra and mysql pull from the same .csv)
- do you need the references to 'data.json' in the seed script? I couldn't confirm it exists, and it looks like the script just runs the INFILE command for your .csv

ruparocks commented on database/cassandra/createReviews.js:
```diff
@@ -0,0 +1,30 @@
+  review.id = id;
+  review.avatar = `https://sdc-reviews-avatars.s3.us-east-2.amazonaws.com/${images[faker.random.number({ min: 0, max: 404 })]}`;
+  review.date = faker.date.past(10);
+  review.entry_id = faker.random.number({ min: 1, max: 1000000 });
```
Is the entry_id a reference to the property id?

willmainwaring replied:
Yes, it is. I realized I should set the max value based on the number of reviews I'm creating, so I replaced it with a variable.

ruparocks commented on database/mysql/createReviews.js:
```diff
@@ -0,0 +1,28 @@
+};
+
+const saveReviews = (start, end) => {
+	for (var i = start; i < end; i++) {
```
You could try to write more than one record to the csv at a time, just a recommendation.

ruparocks commented on database/mysql/schema.sql:
```diff
@@ -6,9 +6,10 @@ USE airbnb;
-  avatar VARCHAR(25) NOT NULL,
-  numDaysAgo INT NOT NULL,
+  avatar VARCHAR(75) NOT NULL,
+  date  VARCHAR(30) NOT NULL,
```
Date is a varchar? Why is this not a Datetime?

willmainwaring replied:
I was only interested in the year/month/day in the date, and I was having some trouble figuring out which date format would work with mySQL, so I converted it to a string with a plan to slice off the first x characters.

I think you're right, though, so I'm changing this to datetime.

ruparocks reviewed (commented):
Clean code. I left a few questions if you can comment back and I understand your choices, I will approve the PR.

willmainwaring commented:
@keenanthomson 

Goods suggestions! I've simplified to one `createReviews` that accepts an id as an argument and will conditionally create a matching key for each review.

Also, `seed.js` was an old file, so I deleted it.

Commit:
refactor createReviews.js

Commit:
handle dates in YYYY-MM-DD format

Commit:
add variable to seed.sql to use same csv for both dbs
